server/conf/messages

```diff
@@ -216,7 +219,6 @@ validation.streetRequired=Please enter valid street name and number.
 validation.cityRequired=Please enter city.
 validation.currencyMisformatted=Currency must be one of the following formats: 1000 1,000 1000.30 1,000.30
 validation.stateRequired=Please enter state.
-validation.zipcodeRequired=Please enter valid ZIP code.
```

Comment:
This was unnecessary since the current regexp for matching a zip-code was a more specific error message anyway.